.github/workflows/build-distribute.yml

```
@@ -0,0 +1,136 @@
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        
+      - name: Get version from build.gradle 
+        id: get-version 
+        run: | 
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:33: Double quote to prevent globbing and word splitting

(shellcheck)